### PR TITLE
Improve Order and OrderedItem models

### PR DESCRIPTION
**Pull request by artursmet**
* Added usable fields for DeliveryGroup (shipping method, tracking number, last_updated)
* Added `total` field in `Order` model
* Fixed error handling in `ShipGroupForm`
* Stock record and stock location is now saved in `OrderedItem` model

We should also find some place, to display parcel tracking number in order dashboard

### CODE REVIEW

File: saleor/dashboard/order/forms.py [hunk 1]
```diff
@@ -144,7 +144,14 @@ def save(self):
 class ShipGroupForm(forms.ModelForm):
     class Meta:
         model = DeliveryGroup
-        fields = []
+        fields = ['shipping_method', 'tracking_number']
+
+    def __init__(self, *args, **kwargs):
+        super(ShipGroupForm, self).__init__(*args, **kwargs)
+        self.fields['shipping_method'].widget.attrs.update(
```
Comment (patrys) [inline saleor/dashboard/order/forms.py]: If you want to expose that in the UI, you need to use the appropriate function to determine a list of allowed choices.

File: saleor/dashboard/templates/dashboard/order/modal_ship_delivery_group.html [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
-<div class="modal-content">
-    <h5>{% trans "Ship delivery group " %}#{{ group.pk }}</h5>
+<form id="ship-delivery-group-form" method="post" role="form" action="{% url 'dashboard:ship-delivery-group' order_pk=order.pk group_pk=group.pk %}" class="form-async">
+    {% csrf_token %}
 
-    <p>{% trans "Ship selected delivery group to the following address:" %}</p>
-    {% include 'dashboard/includes/_address.html' with address=group.order.shipping_address only %}
-</div>
+    <div class="modal-content">
+        <h5>{% trans "Ship delivery group " %}#{{ group.pk }}</h5>
```
Comment (patrys) [inline saleor/dashboard/templates/dashboard/order/modal_ship_delivery_group.html]: Is showing a DB identifier here improving usability? Did I want to ship group `#1653` or `#1635`?

**Comment by artursmet** [comment]
@patrys Fixed as discussed yesterday. Please review.

File: saleor/order/models.py [hunk 3]
```diff
@@ -64,6 +64,10 @@ class Order(models.Model, ItemSet):
                                              editable=False)
     token = models.CharField(
         pgettext_lazy('Order field', 'token'), max_length=36, unique=True)
+    total = PriceField(
```
Comment (patrys) [inline saleor/order/models.py]: Can we have explicit "total_net" and "total_tax"?

**Pull request merged**